### ISSUE TITLE
fractions import error in python 3.9

### ISSUE THREAD

**Issue by jonahweissman** [bnlucas/python-basehash#12]
If I run
```
docker run -it python:3.9 /bin/bash
# pip install basehash
# python -c "import basehash; basehash.base36(2)"
```
I get:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.9/site-packages/basehash/__init__.py", line 84, in __init__
    super(base36, self).__init__(BASE36, length, generator)
  File "/usr/local/lib/python3.9/site-packages/basehash/__init__.py", line 31, in __init__
    self.prime = next_prime(int((self.maximum + 1) * self.generator))
  File "/usr/local/lib/python3.9/site-packages/basehash/primes.py", line 292, in next_prime
    while not is_prime(n):
  File "/usr/local/lib/python3.9/site-packages/basehash/primes.py", line 261, in is_prime
    if gcd(n, 510510) > 1:
  File "/usr/local/lib/python3.9/site-packages/basehash/primes.py", line 38, in gcd
    from fractions import gcd
ImportError: cannot import name 'gcd' from 'fractions' (/usr/local/lib/python3.9/fractions.py)
```

It works fine if I use Python 3.7 or 3.8.

**Comment by jonahweissman** [comment]
Very sorry, looks like this was already fixed in #11. Any chance you could release a new version?

**Comment by bnlucas** [comment]
https://pypi.org/project/BaseHash/3.0.9

Sorry about this. Thought I had updated PyPi back when this was merged.
